cli/azd/cmd/config.go

```diff
@@ -457,9 +457,7 @@ func (a *configListAlphaAction) Run(ctx context.Context) (*actions.ActionResult,
 		return nil, err
 	}
 
-	featureKeys := maps.Keys(features)
-	slices.Sort(featureKeys)
-
+	featureKeys := slices.Sorted(maps.Keys(features))
```

Comment:
As part of being added to the stdlib, `maps.Keys` now returns an iterator and `slices.Sorted` (and `slices.SortedFunc`) and `slices.Collect` collect the values into a (sorted) slice.